pages/linux/yaourt.md

```diff
@@ -0,0 +1,23 @@
+# yaourt
+
+> Arch Linux utility for building packages from the Arch User Repository (AUR)
+
+- synchronize and update all packages, including AUR packages which will be re-compiled
+
+`yaourt -Syua`
+
+- install a new package
+
+`yaourt -S package-name`
+
+- remove a package and its dependencies
+
+`yaourt -Rs package-name`
+
+- search the package database for a keyword
```

Comment:
You might want to mention AUR here too, since you did that for the first example :)